react/src/components/search-app/SearchResultsComponent.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+
+export default class SearchResultsComponent extends React.Component {
```

Comment:
I personally like that there's a separation for a search bar + suggestions and a results component. Also I'm not completely sure I understand the proposed change by @lustoykov . As far as I understand we have two types of queries, autosuggest search and standard search. Do I understand correctly that you want to completely remove the standard search results and only leave the `SearchBarComponent` with the autosuggestions? Or do you mean, that instead of express delivering the search results through `dataForReact` we make an API call, in that case this API call could still be done through the `SearchResultsComponent`?